### PR TITLE
 [DEV-11525] Update recipient states endpoint [hotfix][qat]

### PR DESCRIPTION
**Pull request by aguest-kc**
Duplicate of #4204 for QAT

### CODE REVIEW

**Comment by aguest-kc** [comment]
:stop: Do not merge QAT into this branch until after the Staging and Prod PRs have been merged :stop:

**Review by collinwr — APPROVED** [review]

**Pull request merged**
